pkg/dockerutil/dockerutils.go

```diff
@@ -331,6 +332,22 @@ func CheckDockerVersion(versionConstraint string) error {
 	return nil
 }
 
+// CheckDockerCompose determines if docker-compose is present and executable on the host system. This
+// relies on docker-compose being somewhere in the user's $PATH.
+func CheckDockerCompose() error {
+	executableName := "docker-compose"
+	if runtime.GOOS == "Windows" {
+		executableName = "docker-compose.exe"
+	}
+
+	_, err := exec.LookPath(executableName)
```

Comment:
Let's actually check the docker-composer version. The version that ships with Ubuntu normal packages is too old to do v3 docker-compose.yml. So please figure out the version where they changed it to recognize "version" key, and check to make sure we have a good-enough version.

Reply:
Sounds like the version we want is [1.10.0](https://docs.docker.com/release-notes/docker-compose/#1100-2017-01-18).

Reply:
Fixed in 34d7351